File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/playback/SeekControl.java
/**
 *
 * @file
 *
 * @brief Seek controller interface
 *
 * @author vitamin.caig@gmail.com
 *
 */

package app.zxtune.playback;

import app.zxtune.TimeStamp;

public interface SeekControl {
  
  TimeStamp getDuration();

  TimeStamp getPosition() throws Exception;

  void setPosition(TimeStamp position) throws Exception;
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/playback/SeekControl.kt
/**
 *
 * @file
 *
 * @brief Seek controller interface
 *
 * @author vitamin.caig@gmail.com
 */
package app.zxtune.playback

import app.zxtune.TimeStamp

interface SeekControl {
    val duration: TimeStamp
    var position: TimeStamp
}
